diesel/src/mysql/connection/url.rs

```diff
@@ -8,6 +8,9 @@ use std::ffi::{CStr, CString};
 
 use crate::result::{ConnectionError, ConnectionResult};
 
+#[cfg(not(windows))]
```

Comment:
all of these `cfg` flags can be retired now, yeah?

Reply:
yes.